fastlane/Fastfile

```diff
@@ -638,16 +656,19 @@ REPOSITORY_NAME="WordPress-Android"
   #####################################################################################
   desc "Builds an app bundle"
   lane :build_for_translation_review do | options |
+    android_download_translations(
+      glotpress_url: 'https://translate.wordpress.com/projects/wporg/apps/android',
+      status_filter: ['current', 'waiting', 'fuzzy'],
+      locales: SUPPORTED_LOCALES,
```

Comment:
For this one you need to only pass the subset of locales that is supported by the GlotPress project you use as URL, which seems different from the main one for normal build.

This means that you need to only pass the subset of `ALL_LOCALES` that matches what is listed in `tools/review-language_codes`.

What I suggest, similar to previous comment, is to have the `ALL_LOCALES` array have an additional key, e.g. `translator: true/false`, for each entry, and then use `ALL_LOCALES.select { |h| h[:translator] }` to derive the list to pass as parameter to this call site.

Another approach could be to list the glotpress locales and select entries matching those:
```ruby
translator_locales = %w[de es fr id it ja ko nl ru sv zh-cn zh-tw tr he pt ar]
translator_locales_map = ALL_LOCALES.select { translator_locales.include?(h[:glotpress]) }
```
(and then pass translator_locales_map variable to the locales: parameter)

Reply:
Agreed! 👍  I'll opt to just pull out the subset using your second suggestion here, thanks @AliSoftware

Reply:
The list I used was `%w[de es fr id it ja ko nl ru sv zh-cn zh-tw tr he pt ar pl th sr vi]` which I think has all the locales that would be translated, let me know if you had any other questions/feedback on this, thanks!